test/cqlpy/test_lwt.py

```diff
@@ -103,6 +108,40 @@ def test_lwt_with_batch_conflict_2(cql, table1):
     for r in rs:
         assert r.applied == False
 
+# Moreover, there are cases where Cassandra prevents mixing
+# IF EXISTS with different conditions such as IF r=1, despite
+# the fact that the conditions are not contradictory.
+def test_lwt_with_batch_conflict_3(cql, table1, scylla_only):
+    p = unique_key_int()
+    cql.execute(f'INSERT INTO {table1}(p,c,r) VALUES ({p},1,1)')
+    rs = cql.execute(f'BEGIN BATCH UPDATE {table1} SET r=10 WHERE p={p} AND c=1 IF r=1; UPDATE {table1} SET r=20 WHERE p={p} AND c=1 IF EXISTS;  APPLY BATCH;')
+    # Cassandra fails with: "Cannot mix IF conditions and IF EXISTS for the same row"
+    for r in rs:
+        assert r.applied == True
+
+# During a static column update, the clustering key is not required in the WHERE clause.
+# A batch composed of a query that updates a static column and a query that inserts
+# a new row is allowed and will be applied successfully.
+def test_lwt_with_batch_conflict_4(cql, table1):
+    p = unique_key_int()
+    cql.execute(f'INSERT INTO {table1}(p,c,r,s) VALUES ({p},1,1,1)')
+    rs = cql.execute(f'BEGIN BATCH UPDATE {table1} SET s=NULL WHERE p={p} IF EXISTS; INSERT INTO {table1}(p,c,r) VALUES ({p},2,2) IF NOT EXISTS; APPLY BATCH;')
+    for r in rs:
+        assert r.applied == True
```

Comment:
I don't understand why this test shows any sort of "conflict". I'm not against this test though I'm not sure I understand why it's useful - but perhaps you should reconsider its name?